src/redux/reducers/lines.js

```diff
@@ -32,6 +32,18 @@ export default function reducer(state = initialState, action = {}) {
       action.result.result.verses.forEach((ayahId) => {
         const ayah = ayahs[ayahId];
 
+        let pauses = 0
```

Comment:
How about moving this into [`buildSegements`](https://github.com/quran/quran.com-frontend/blob/master/src/helpers/buildSegments.js)?

Reply:
Or maybe fix the data...